app/auth/routes.py

```diff
@@ -0,0 +1,95 @@
+"""
+Authentication settings for the BucketList API.
+
+Password verification and user registration takes place here.
+"""
+from flask import g, jsonify, request
+from flask_httpauth import HTTPBasicAuth
+
+from . import authe
+from app import errors
+from app.decorators import json
+from app.models import User
+
+auth = HTTPBasicAuth()
+
+
+@auth.verify_password
+def verify_password(token, password):
+    """
+    Verify token.
```

Comment:
Why not call it verify_token instead? I think there is a class HTTPTokenAuth() that would serve you better.